pipelines/optimus/README.md

```diff
@@ -60,7 +60,12 @@ Optimus can be deployed using [Cromwell](https://software.broadinstitute.org/wdl
 
 ## Inputs
 
-Optimus pipeline inputs are detailed in a JSON file, such as in this [example](https://github.com/HumanCellAtlas/skylab/blob/master/pipelines/optimus/example_test_inputs.json). 
+Optimus pipeline inputs are detailed in JSON format configuration files. There are three sample configuration files available if you are interested in running the pipeline. 
+*  [human_v2_example](human_v2_example.json): A human 10x v2 single-cell dataset
```

Comment:
Either say which dataset it is here or say 'an example human 10x v2...'

Reply:
Done